src/api/app/views/webui2/webui/request/_review_tab.html.haml

```diff
@@ -1,14 +1,15 @@
 = form_tag(action: 'modify_review') do
   = hidden_field_tag('request_number', bs_request.number)
   = hidden_review_payload(review)
-  %p
-    = user_name_with_icon(User.find_by(login: review[:who]))
-    requested:
-  .ml-4
+  - if review[:creator]
     %p
-      #{simple_format(review_request_reason(bs_request, review))}
+      = user_name_with_icon(User.find_by(login: review[:creator]))
+      requested:
+    .ml-4
+      %p
```

Comment:
The interpolation is not needed here, so it should be

`%p= simple_format(review[:reason])`